hw/top_earlgrey/formal/conn_csvs/ast_rstmgr.csv

```diff
@@ -0,0 +1,21 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Run these checks with:
+#  ./util/dvsim/dvsim.py hw/top_earlgrey/formal/chip_conn_cfg.hjson
+
+,NAME,SRC BLOCK,SRC SIGNAL,DEST BLOCK,DEST SIGNAL,,,,,,
+
+#################################
+# Signals from Rstmgr to AST
```

Comment:
Some connections below may be replicated in connections I just added. Perhaps it is better to avoid the replication, in which case it may be best to keep them in the AST csvs, and keep the IP connectivity more independent of the actual top?

Same thing at least for clkmgr connectivity.

Reply:
I went through the clock and reset manager tests and consolidated them in the last two commits.